### ISSUE TITLE
ix select doesn't get the data on the first click on dropdown item

### ISSUE THREAD

**Issue by AnnaGrahl** [siemens/ix#1377]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

We wanted to update to the latest ix version. We faced some issues regarding the ix select. 
We noticed this with our cypress tests which failed after the update. 

After the update, we need to select two times the item in the dropdown menu. On the first click the value is undefined. Also in the code example i provided you, you notice, that the value is selected, after two clicks, the first is the value before.
![image](https://github.com/siemens/ix/assets/153725916/677a179c-5581-49d7-8323-34a27fbc6023)

![image](https://github.com/siemens/ix/assets/153725916/75a2aa1c-3984-48c1-b60a-10d5285c9572)

![image](https://github.com/siemens/ix/assets/153725916/9907f17d-c698-4624-875d-334b3dee08b6)

### What type of frontend framework are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.4.0

### Code to produce this issue.

```shell
import { Component } from '@angular/core';

@Component({
  selector: 'app-example',
  template: `
    <ix-select
    (itemSelectionChange)="onGatewaySelectionChange($event)">
      <ix-select-item label="Item 1" value="1"></ix-select-item>
      <ix-select-item label="Item 2" value="2"></ix-select-item>
      <ix-select-item label="Item 3" value="3"></ix-select-item>
      <ix-select-item label="Item 4" value="4"></ix-select-item>
    </ix-select>
  `,
})
export default class Select {
  value = '1';

  onGatewaySelectionChange(event: Event): void {
    const select = event.target as HTMLSelectElement;
    console.log('Selected gateway:', select.value);
  }
}
```

**Comment by danielleroux** [comment]
@AnnaGrahl You can workaround this issue by using the event detail:

```
  onGatewaySelectionChange(event: Event): void {
    const select = event as CustomEvent;
    console.log('Selected gateway:', select.detail);
  }
```

**Comment by m7mdyahia** [comment]
was this solved in 2.4.1?

**Comment by Domi1980** [comment]
@m7mdyahia Doesn't seem to be the case. I mistakenly merged 2.4.1 into our codebase and observed the same issue.
But the workaround posted by @danielleroux seems to fix it for now.
